lib/realtime/repo.ex

```diff
@@ -8,22 +8,62 @@ defmodule Realtime.Repo do
     {:ok, repo} = [name: nil, pool_size: 2] |> Keyword.merge(config) |> Realtime.Repo.start_link()
 
     try do
-      Realtime.Repo.put_dynamic_repo(repo)
+      __MODULE__.put_dynamic_repo(repo)
       callback.(repo)
     after
-      Realtime.Repo.put_dynamic_repo(default_dynamic_repo)
+      __MODULE__.put_dynamic_repo(default_dynamic_repo)
```

Comment:
Question, maybe we can skip the namespacing altogether? Same for `start_link` above.

```suggestion
      put_dynamic_repo(default_dynamic_repo)
```

Reply:
makes sense 👍 will change